rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -169,10 +169,30 @@ components:
       type: string
       minLength: 1
       maxLength: 256
-    Metacard:
+    Metadatas:
       description: >
-        A metacard.xml attachment that is sent along with a product to enhance the product's discoverability. The
-        current maximum is the character equivalent of 10GB.
+        A list of metadata objects that a client can pass along with a product to enrich indexing and query
+        capabilities.
+      type: array
+      items:
+        $ref: '#/components/schemas/Metadata'
+    Metadata:
+      description: An object that contains the metadata type and the metadata file.
+      type: object
+      properties:
+        metadataType:
+          $ref: '#/components/schemas/MetadataType'
+        metadataContent:
+          $ref: '#/components/schemas/MetadataContent'
+    MetadataType:
+      description: >
+        The type of metadata that is being sent.
+      type: string
+      minLength: 1
+      maxLength: 256
+    MetadataContent:
+      description: >
+        The metadata file attachment. The current maximum is the character equivalent of 10GB.
       type: string
       format: binary
```

Comment:
❓ I think @clockard Had a question about ether this has to be binary or if there are cases where it will be text based. If the later than we might want to account for text types too to avoid encoding text for no reasons.

Reply:
Binary should cover all cases. Until we prove binary is unworkable in some situations, I would like to stick with it.

Reply:
Since we handle the binary content as a MultipartFile on the Spring side, the client can still specify Content-Type of `text-plain` for text.

Reply:
✏️ It still is weird that the spec says binary

Reply:
Now that we create the interface manually we could omit this format property, but I want to leave it in to minimize changes between the API and the spec.